sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_response_handlers.py

```diff
@@ -29,7 +29,7 @@ def prepare_us_receipt(response):
     for page in document_result:
         if page.fields is None:
             receipt = USReceipt(
-                page_range=PageRange(first_page=page.page_range[0], last_page=page.page_range[1]),
+                page_range=FormPageRange(first_page=page.page_range[0], last_page=page.page_range[1]),
```

Comment:
One question I have about this change: shouldn't we also update prepare_us_receipt to return a list of RecognizedReceipt instead of USReceipt (line 31)?

Reply:
thats a good question - but at this point since we support only us receipt - we will return the USReceipt - the RecognizedReceipt is for *documentation* purposes so that we keep it more open for future - it's something we will be able to change internally later

Reply:
this might be too much for the future, but should we add the ability in RecognizedReciept to map to which of its children it should be?
i.e., adding a dict of receipt-locale to child type {"en-us": USReceipt}, so we could pass in this into to RecognizedReceipt and get out a USReceipt in this case.

@kristapratico what are your thoughts on this, I wasn't in the phone call so not sure what you guys decided for now

Reply:
Yeah I think that makes sense for the future. v2 will GA with only US sales receipts so we'll add that implementation detail in v2.1.